Website/Packages/datax-pipeline/src/modules/flowDefinition/flowReducer.js

```diff
@@ -49,6 +55,7 @@ export default (state = INITIAL_FLOW_STATE, action) => {
             return Object.assign({}, INITIAL_FLOW_STATE, flow, {
                 isNew: false,
                 isDirty: false,
+                selectedFlowBatchInputIndex: flow.input.batch && flow.input.batch.length > 0 ? 0 : undefined,
```

Comment:
bug: use selectedBatchIndex here instead

Reply:
selectedFlowBatchInputIndex is correct. this is an index to support multiple inputs. and selectedBatchIndex is an index for the batch job list.